src/viewPanel/sessionWidget.ts

```diff
@@ -50,6 +50,11 @@ export class SessionWidget extends BoxPanel {
     this._tabPanel.topBar.addRequested.connect(() => {
       this._model.addTab();
     });
+    this._tabPanel.topBar.tabCloseRequested.connect((tab, arg) => {
+      arg.title.owner.close();
```

Comment:
It could be nice to add a confirmation popup, we can easily click on the cross when switching tab.